grafana/meta/collectd.yml

```diff
@@ -0,0 +1,10 @@
+{%- if pillar.grafana.server is defined %}
```

Comment:
If a node is classified with 'service.grafana.client', there's a grafana key in pillar but no grafana server running.